src/Languages/Editor/Impl/Microsoft.Languages.Editor.csproj

```diff
@@ -156,6 +156,10 @@
       <Project>{8d408909-459f-4853-a36c-745118f99869}</Project>
       <Name>Microsoft.Common.Core</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\Windows\Core\Impl\Microsoft.Windows.Core.csproj">
```

Comment:
Ok, though I'm not sure about namespace. Extracted code is indeed platform-specific.